pkg/errcode/errcode.go

```diff
@@ -0,0 +1,31 @@
+package errcode
+
+import "fmt"
+
+type errCode struct {
+	code int
+	text string
+}
+
+func (e *errCode) Error() string {
+	return e.text
+}
+
+// Return error code value or NotErrCodeType if variable isn't of type errCode
```

Comment:
Function/Method description comments should start with the method name, e.g
(golang documentation convention)

`// GetCode returns error code value .....`

lets adjust for this package